plugins/modules/docker_config.py

```diff
@@ -107,16 +104,15 @@
 author:
   - Chris Houseknecht (@chouseknecht)
   - John Hu (@ushuz)
-'''
-
-EXAMPLES = '''
+"""
 
+EXAMPLES = r"""
 - name: Create config foo (from a file on the control machine)
   community.docker.docker_config:
     name: foo
-    # If the file is JSON or binary, Ansible might modify it (because
-    # it is first decoded and later re-encoded). Base64-encoding the
-    # file directly after reading it prevents this to happen.
+  # If the file is JSON or binary, Ansible might modify it (because
+  # it is first decoded and later re-encoded). Base64-encoding the
+  # file directly after reading it prevents this to happen.
```

Comment:
@russoz this is another thing thats get changed to the worse (IMO): comments in YAML dictionaries are *sometimes* unindented one level.
```suggestion
    # If the file is JSON or binary, Ansible might modify it (because
    # it is first decoded and later re-encoded). Base64-encoding the
    # file directly after reading it prevents this to happen.
```